src/containers/Auth/index.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { auth } from '../../store/actions';
+
+class Auth extends Component {
+  componentDidMount() {
```

Comment:
We can use the useEffect() hook. See: 
https://dev.to/trentyang/replace-lifecycle-with-hooks-in-react-3d4n

Reply:
The useEffect() hook only works in pure React components, for example:
const auth = () => {...}
Instead of:
class Auth extends Component {...}

I'm not using a pure React component on purpose, so I can use Redux